documentation/man/features/kw-manager.rst

```diff
@@ -0,0 +1,93 @@
+===============================================
+kw-manager - Manage groups of emails 
+===============================================
+
```

Comment:
I see... you created a single commit for this. Ideally, you want to grow the documentation in the PR as you add new things, but for simplicity's sake, you can ignore my previous comments about the documentation and just keep this final commit with that.